pkg/console/operator/operator.go

```diff
@@ -157,6 +177,24 @@ func (c *consoleOperator) handleSync(operatorConfig *operatorsv1.Console, consol
 		// handled below
 	case operatorsv1.Unmanaged:
 		logrus.Println("console is in an unmanaged state.")
+		v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
```

Comment:
Ok, so if I set the `Console` to `Unmanaged`, all of theses statuses will be applied.  When I change it back to `Mananaged`, or to `Removed` or `Force`, we will need to clear these or update them to something else.

Reply:
That should already be happening since you should have already had something that was setting these under steady state (ie on resync as well as on resource updates).

Reply:
We don't yet, this is the first PR to do anything with Conditions.